src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -487,7 +486,9 @@ public async Task<IActionResult> SaveAnswers(PageViewModel vm, Guid applicationI
             
             if (page == null)
             {
-                return RedirectToAction("TaskList", new { applicationId = applicationId });
+                return sequenceId != null ?
```

Comment:
line 435: var sequenceId = int.Parse(vm.SequenceId); // TODO: SequenceId should be an int, not a string

Either that above is a now a bug as it can be null. Or the sequenceId  != null check here will always be true